src/Client.ts

```diff
@@ -52,4 +56,41 @@ export class Client {
       token: accessToken,
     });
   }
+
+  public async getReports(query?: GetStationsQuery): Promise<Array<Report>> {
```

Comment:
Should this be GetReportsQuery? ^^

Reply:
We named the above query methods `getData` and `getStations`. It returns an `Array<Report>` so I think the name works. We could change them to `queryData`, `queryStations`, & `queryData`?

Reply:
@dcavanagh I think he means the type of the first argument.